### PR TITLE
WIP easteregg/add_vpn_page

### PR DESCRIPTION
**Pull request by easteregg**

### CODE REVIEW

File: src/sass/static/pages.scss [hunk 1]
```diff
@@ -1207,3 +1207,75 @@ body #not_authenticated_financial ul.checked > li {
         }
     }
 }
+
+.vpn-app {
+    .steps.steps-vertical {
+        @media (max-width: 480px) {
```
Comment (negarn) [inline src/sass/static/pages.scss]: generally we put media queries at the bottom of the class

File: src/sass/static/pages.scss [hunk 2]
```diff
@@ -1207,3 +1207,75 @@ body #not_authenticated_financial ul.checked > li {
...
+            flex-direction: column;
+
+            .step:not(:first-child) {
+                margin-top: 2rem;
+            }
+        }
+        max-width: 603px;
```
Comment (negarn) [inline src/sass/static/pages.scss]: exactly 603 😅

File: src/sass/static/pages.scss [hunk 3]
```diff
@@ -1207,3 +1207,75 @@ body #not_authenticated_financial ul.checked > li {
...
+            transform: scale(0.85);
+        }
+    }
+    .column {
+        @media screen and (max-width: 480px) {
+            padding: 0;
+        }
+    }
+    .vertical-divider {
+        @media screen and (max-width: 480px) {
```
Comment (negarn) [inline src/sass/static/pages.scss]: same here

File: src/templates/app/user/security.jsx [hunk 4]
```diff
@@ -54,6 +54,10 @@ const Security = () => (
                 <Column url='authorised_appsws' image='applications' header={it.L('Authorised Applications')} text={it.L('Manage your authorised applications.')} />
 
                 <Column url='two_factor_authentication' image='2fa' header={it.L('Two-Factor Authentication')} text={it.L('Enable two-factor authentication for an extra layer of security.')} />
+
+                <Column url='cloudflare_dns' image='1111' header={'1.1.1.1'} text={it.L('Use Cloudflare’s public DNS service for faster and more secure browsing.')} />
```
Comment (negarn) [inline src/templates/app/user/security.jsx]: why adding this again?

File: src/templates/app/user/security.jsx [hunk 5]
```diff
@@ -54,6 +54,10 @@ const Security = () => (
...
+
+                <Column url='vpn_app' image='vpn' header={'VPN'} text={it.L('Establish a secure Internet connection and protect your privacy.')} />
```
Comment (negarn) [inline src/templates/app/user/security.jsx]: should the header be `VPN App` similar to the tab title?

File: src/templates/app/user/security/vpn_app.jsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+const Step = ({
+    circle_number,
+    text,
+    imgSrc,
```
Comment (negarn) [inline src/templates/app/user/security/vpn_app.jsx]: please also use underline separator for this

File: src/templates/app/user/security/vpn_app.jsx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+        <div className='circle'>{circle_number}</div>
+        <div className='content gr-row gr-padding-10 gr-row-align-between'>
+            <p>{text}</p>
+            <img className='vpn-app-icon' src={imgSrc} />
+        </div>
+    </div>
+);
+const VPNApp = () => (
+    <div className='vpn-app'>
+        <h1>VPN app</h1>
```
Comment (negarn) [inline src/templates/app/user/security/vpn_app.jsx]: we don't need translation for this?

File: src/templates/app/user/security/vpn_app.jsx [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+                <Step
+                    circle_number='2'
+                    text={it.L('Activate the VPN service on the app')}
+                    imgSrc={it.url_for('images/pages/vpn/ic-activate.svg')}
+                />
+                <Step
+                    circle_number='3'
+                    text={it.L('Browse and trade securely on your preferred device')}
+                    imgSrc={it.url_for('images/pages/vpn/ic-browse.svg')}
+                />
```
Comment (negarn) [inline src/templates/app/user/security/vpn_app.jsx]: not sure if it's better to dynamically increase the circle numbers or to pass them in

File: src/templates/app/user/security/vpn_app.jsx [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+const Step = ({
+    circle_number,
+    text,
+    imgSrc,
+}) => (
+    <div className='step'>
+        <div className='circle'>{circle_number}</div>
+        <div className='content gr-row gr-padding-10 gr-row-align-between'>
+            <p>{text}</p>
+            <img className='vpn-app-icon' src={imgSrc} />
```
Comment (negarn) [inline src/templates/app/user/security/vpn_app.jsx]: why not put `it.url_for('images/pages/vpn/[...].svg')` here and just pass in the image name?

Comment (negarn) [reply src/templates/app/user/security/vpn_app.jsx]: it's up to you, it's more easy to find where images are used if we don't do what I mentioned.

File: src/templates/app/user/security/vpn_app.jsx [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>
+                </div>
+                <div className='vertical-divider' />
+                <div className='column gr-gutter-left'>
+                    <div className='gr-padding-10'>
+                        <a
+                            className='button'
+                            href='https://play.google.com/store/apps/details?id=com.binary.outline.android.client'
+                            rel='nofollow noopener'
+                            title={it.L('Download Binary VPN app For Android')}
```
Comment (negarn) [inline src/templates/app/user/security/vpn_app.jsx]: please also add the download attribute here. do we need to open it in a new tab @ashkanx ?

Comment (ashkanx) [reply src/templates/app/user/security/vpn_app.jsx]: Android link should open in new tab

**Pull request merged**